package/network/config/ltq-vdsl-app/files/dsl_control

```diff
@@ -219,7 +219,29 @@ start_service() {
 	esac
 
 	local annexgpio="/sys/class/gpio/annex"
-	if [ -d "${annexgpio}a" ] && [ -d "${annexgpio}b" ]; then
+	local dsl_en_gpio="/sys/class/gpio/dsl_en"
+	if [ -d "${dsl_en_gpio}" ]; then
+		echo 1 > "${dsl_en_gpio}/value"
+	fi
```

Comment:
I do not really see what these changes are good for.

Please have them in a separate commit _before_ adding device support and explain why they are needed.

Reply:
Note the _before_ in the previous comment.